### PR TITLE
[Dart Jaguar] manage nullable definition

### PR DESCRIPTION
**Pull request by jaumard**
also update deps

### CODE REVIEW

File: samples/client/petstore/dart-jaguar/openapi_proto/docs/UserApi.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ try {
 
 Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
- **body** | [**List&lt;User&gt;**](User.md)| List of user object | 
+ **body** | [**List&lt;User&gt;**](List.md)| List of user object |
```
Comment (jaumard) [inline samples/client/petstore/dart-jaguar/openapi_proto/docs/UserApi.md]: not sure what append here as I didn't touch that... a regression @wing328 ?

Comment (wing328) [reply samples/client/petstore/dart-jaguar/openapi_proto/docs/UserApi.md]: Looks like you're not using the latest master ...

**Comment by wing328** [comment]
I'll merge this into master and update the samples.

**Comment by wing328** [comment]
Samples updated via 93aedcf3d5

**Comment by jaumard** [comment]
Thanks ! :) strange because I rebase master branch before doing the PR...

**Comment by wing328** [comment]
@jaumard thanks for the PR, which has been included in the v4.1.1 release: https://twitter.com/oas_generator/status/1165944867391860737

**Pull request merged**